homeassistant/components/advantage_air/cover.py

```diff
@@ -113,3 +123,37 @@ async def async_set_cover_position(self, **kwargs: Any) -> None:
                     }
                 }
             )
+
+
+class AdvantageAirThingCover(AdvantageAirThingEntity, CoverEntity):
+    """Representation of Advantage Air Cover controlled by MyPlace."""
+
+    _attr_supported_features = CoverEntityFeature.OPEN | CoverEntityFeature.CLOSE
+
+    def __init__(
+        self,
+        instance: dict[str, Any],
+        thing: dict[str, Any],
+        device_class: CoverDeviceClass,
+    ) -> None:
+        """Initialize an Advantage Air Things Cover."""
+        super().__init__(instance, thing)
+        self._attr_device_class = device_class
+
+    @property
+    def is_closed(self) -> bool:
+        """Return if cover is fully closed."""
+        return self._data["value"] == 0
+
+    @property
+    def current_cover_position(self) -> int:
+        """Return covers current position as a percentage."""
+        return self._data["value"]
```

Comment:
```suggestion
```

https://github.com/home-assistant/core/blob/00191ace6c0ba921a4144ad2c8722a0837533e2d/homeassistant/components/cover/__init__.py#L303

If this property is set it will imply that set position is supported when its not